src/common/interop/shared_constants.h

```diff
@@ -13,6 +13,8 @@ namespace CommonSharedConstants
     // Path to the event used by PowerLauncher
     const wchar_t POWER_LAUNCHER_SHARED_EVENT[] = L"Local\\PowerToysRunInvokeEvent-30f26ad7-d36d-4c0e-ab02-68bb5ff3c4ab";
 
+    const wchar_t SEND_RUN_SETTINGS_TELEMETRY_EVENT[] = L"Local\\PowerToysRunInvokeEvent-638ec522-0018-4b96-837d-6bd88e06f0d6";
```

Comment:
Let's rename it `RUN_SEND_SETTINGS_TELEMETRY_EVENT` since we are going to use this approach for all modules, and since we might have more events for one module, it's better to organize them by module name.
We will also have to rename `POWER_LAUNCHER_SHARED_EVENT` but not in this PR.

Reply:
done